### ISSUE TITLE
wrong padding of AdaptiveFeatureGenerator

### ISSUE THREAD

**Issue by htzheng** [microsoft/CoCosNet#11]
Hi, I noticed that for AdaptiveFeatureGenerator, the encoder is implemented using padding of Conv,
`self.layer1 = norm_layer(nn.Conv2d(opt.spade_ic, ndf, kw, stride=1, padding=pw))`
However, the warping images have strong artifacts at boundry.
![image](https://user-images.githubusercontent.com/8148851/87875296-55387300-c99e-11ea-81d5-3a4446026fce.png)

Instead, when replacing the padding with reflect padding, the result is significantly improved. 
![image](https://user-images.githubusercontent.com/8148851/87875362-c4ae6280-c99e-11ea-9012-56343e02b894.png)

Because of this, it seems there is a difference between the pretrained model and the testing code. Also, when using reflect padding for training, I often find the training collapses with unknow error after several epochs. Did you notice the same issue?

**Comment by panzhang0212** [comment]
Could you show the error log?

**Comment by htzheng** [comment]
Hi, in the AdaptiveFeatureGenerator, I found feature backbone is extracted without reflect padding,
`self.layer1 = norm_layer(nn.Conv2d(opt.spade_ic, ndf, kw, stride=1, padding=pw))`
Because of this, the warped images have surrounding artifacts.
![image](https://user-images.githubusercontent.com/8148851/90342435-5c22c780-dfd6-11ea-9b07-092aae7be10e.png)

I wonder is there any ways to solve this artifact problem. Thanks.
